MVP/Configs/Environment.swift

```diff
@@ -53,4 +54,11 @@ public enum Environment {
         }
         return leaderboardUrl
       }()
+    
+    static let playersUrl: String = {
+      guard let playersUrl = Environment.infoDictionary[Keys.Plist.playersUrl] as? String else {
+        fatalError("LEADERBOARD not set in plist for this environment")
```

Comment:
A copy and paste error but it still says leaderboard in the error text instead of playersurl